frontend-react/src/senders/hooks/UseSenderSchemaOptions/index.ts

```diff
@@ -0,0 +1,64 @@
+import { useSenderResource } from "../../../hooks/UseSenderResource";
```

Comment:
This is using the new domain structure we discussed:

```
src/
  senders/
    hooks/
      UseSenderSchemaOptions/
        index.ts
        UseSenderSchemaOptions.test.ts
```

@jpandersen87 @etanb Thoughts on this?  One tiny gripe I have is that Intellij doesn't automatically the index.ts file when I type `UseSenderSchemaOptions` -- it opens up the directory instead 😆

Reply:
Can you elaborate on this?

Reply:
Sure!  Here's a little GIF:

![2023-02-08 10 36 26](https://user-images.githubusercontent.com/2421042/217576695-39c4847f-732f-4b9b-98a2-0389cf90833c.gif)

It just opens up the directory in the sidebar rather than the file proper -- I'd have to type `UseSenderSchemaOptions/index.ts` to open the latter.  Not a huge deal, just a bit of an annoyance on my end :P